python/lsst/ctrl/mpexec/dotTools.py

```diff
@@ -223,6 +224,7 @@ def pipeline2dot(pipeline, file, taskFactory=None):
         close = True
 
     print("digraph Pipeline {", file=file)
+    universe = DimensionUniverse.fromConfig()
```

Comment:
Could you move this line to the very beginning of the method? If it raises for some reason I'd prefer not to write anything to a file or not to open a file.